app/controllers/clients_controller.rb

```diff
@@ -5,32 +5,22 @@ class ClientsController < ApplicationController
   before_filter :check_authorized_access, except: [:index, :new]
  
   def index
-    @clients = Client.where("company_id = ?",current_user.company_id)
+    @clients = Client.where(company_id: current_user.company_id)
   end
 
   def new
     @client = Client.new
-    @address = @client.address
-    if @address.present?
-    else
-      @address = Address.new
-    end
+    @address = Address.new
```

Comment:
if `@address` in not used elsewhere, consider creating and and assigning to below line without creating instance variable.